src/mono/wasm/build/WasmApp.targets

```diff
@@ -208,4 +246,38 @@
       <WasmAssembliesFinal Include="@(_WasmAssembliesInternal)" LlvmBitCodeFile="" />
     </ItemGroup>
   </Target>
+
+  <!-- Setup NPM packages for NodeJS publish/builds -->
+  <Target Name="_NodePrep" AfterTargets="_AfterWasmBuildApp" Condition="'$(ForNode)' == 'true'">
+    <!-- If package.json is in the directory copy it to new output directory -->
+    <Copy SourceFiles="package.json"
+        DestinationFolder="$(WasmAppDir)"
+        SkipUnchangedFiles="true" 
+        Condition="Exists('package.json')"/>
+
+    <!-- Otherwise create a new file in the output directory with the defaults -->
+     <WriteLinesToFile
+      File="$(WasmAppDir)\package.json"
+      Lines='{
+  "name": "mono-wasm-test",
```

Comment:
This is weird, is there some reason why we can't have a template file somewhere?

Reply:
Maybe we can, how do template files work?

Reply:
Ok that's a good idea. I'm doing that now